### ISSUE TITLE
Rule Suggestion: Don't allow outline: none

### ISSUE THREAD

**Issue by nzakas** [CSSLint/csslint#138]
A lot of times, outline: none is used to hide the focus rectangle on certain elements. This is bad for accessibility, as sighted keyboard users aren't able to tell which elements have focus on the page. Perhaps a rule should be to disallow outline: none under a new category of "accessibility"?
